{
  "features": [
    {
      "geometry": {
        "coordinates": [
          -97.74141,
          30.39425
        ],
        "type": "Point"
      },
      "properties": {
        "desc": {
          "cause": {
            "EN-US": "Under Investigation",
            "ES-ES": "Investigaci\u00f3n en curso",
            "orig": "No Selection"
          },
          "cluster": false,
          "comments": null,
          "crew_icon": true,
          "crew_status": {
            "EN-US": "One or more crews have been assigned. ",
            "ES-ES": "Uno o m\u00e1s equipos se ha asignado. ",
            "orig": "En Route"
          },
          "cust_a": {
            "val": 9
          },
          "etr": "2025-01-22T10:41:04Z",
          "etr_confidence": null,
          "inc_id": null,
          "n_out": 1,
          "start_time": "2025-01-22T08:24:15Z"
        },
        "id": "7-0",
        "source": "https://kubra.io/cluster-data/103/20d293c6-08fd-41b3-b96b-d2f522c74990/46574920-9097-462c-abbe-ea4e6b2dd93f/public/cluster-1/0231301.json",
        "title": "Area Outage"
      },
      "type": "Feature"
    }
  ],
  "type": "FeatureCollection"
}
